pkg/otlp/metrics/runtime_metric_mappings.go

```diff
@@ -0,0 +1,233 @@
+package metrics
+
+// runtimeMetricMapping defines the fields needed to map OTel runtime metrics to their equivalent
+// Datadog runtime metrics
+type runtimeMetricMapping struct {
+	mappedName string                   // the Datadog runtime metric name
+	attributes []runtimeMetricAttribute // the attribute(s) this metric originates from
+}
+
+// runtimeMetricAttribute defines the structure for an attribute in regard to mapping runtime metrics.
+// The presence of a runtimeMetricAttribute means that a metric must be mapped from a data point
+// with the given attribute(s).
+type runtimeMetricAttribute struct {
+	key    string   // the attribute name
+	values []string // the attribute value, or multiple values if there is more than one value for the same mapping
+}
+
+var goRuntimeMetricsMappings = map[string][]runtimeMetricMapping{
```

Comment:
Every runtime mapping should follow this silent contract - `map[string][]runtimeMetricMapping` --- Can we create this as a try or interface ?